### ISSUE TITLE
Virtual scrolling support

### ISSUE THREAD

**Issue by stevenpila** [vakata/jstree#2528]
Hi,

Based from this issue https://github.com/vakata/jstree/issues/1688.
Is there any updates on the virtualization support?

Thanks you.

**Comment by vakata** [comment]
Yes, v.4 has an "inifinite scroller" renderer, but it has no release yet. Still, you can check it out, but it is not production ready yet.
